contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -230,58 +232,139 @@ NXmpes(NXobject):
             This is the momentum axis to be used for data plotting.
     (NXsample):
       name:
-      chemical_formula:
-        exists: recommended
-        doc: |
-          The chemical formula of the sample. For mixtures use the NXsample_component
-          group in NXsample instead.
-      sample_history(NXnote):
-        exists: recommended
+      (NXsubstance):
         doc: |
-          A descriptor to keep track of the treatment of the sample before entering the
-          photoemission experiment. Ideally, a full report of the previous operations, in
-          any format (NXnote allows to add pictures, audio, movies). Alternatively, a
-          reference to the location or a unique identifier or other metadata file. In the
-          case these are not available, free-text description.
+          For samples containing a single pure substances. For mixtures use the
+          NXsample_component_set and NXsample_component group in NXsample instead.
+        molecular_formula_hill:
+          exists: recommended
+          doc: |
+            The chemical formula of the sample.
       atom_types:
```

Comment:
Is this correct here or should it go to NXsubstance?

Reply:
We don't have atom_types in NXsubstance. atom_types is supposed to contain all elements in the sample (incl. subcomponents), while NXsubstance is just for a single, undividable chemical substance.